src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +209,18 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                if (!(controller instanceof LocalController)) {
+                    // TODO: Make it work for RemoteJenkinsController like in Unix (below)
+                    throw new AssumptionViolatedException("Copying files in Windows is only supported if a LocalController is in use. Test will be skipped.");
+                }
+                addBatchStep("xcopy " + file.getAbsolutePath() + " %cd% /E");
```

Comment:
As per [this commit](https://github.com/armfergom/acceptance-test-harness/commit/4264b16e09412febd736bd185ee95a084d7d5ea0) I'd say that the implementation for *NIX systems (code below) was done such weird in order to avoid issues with special characters in the path.

`xcopy " + file.getAbsolutePath() + " %cd% /E"` seems more weak to me, but yes, it covers 99% of use cases, so it's ok IMO.